website/docs/intro.md

```diff
@@ -0,0 +1,52 @@
+---
+slug: /
+sidebar_position: 10
+---
+
+# Getting Started
+
+`nucypher-ts` is a typescript library to allow developers to interact with core Nucypher functionality within the browser.
+It is in active development, so please be aware that things may change!
```

Comment:
My markdown is a bit rusty, but you probably good to highlight this. (Try the suggestion below before using)

```suggestion
**Note:** `nucypher-ts` is in active development, so please be aware that things may change!
{: .note}
```